pkg/runtime/runtime.go

```diff
@@ -1962,3 +2011,128 @@ func (a *DaprRuntime) startReadingFromBindings() error {
 	}
 	return nil
 }
+
+func (a *DaprRuntime) suspendInboundTraffics() error {
+	var merr error
+
+	for name, binding := range a.inputBindings {
```

Comment:
There is work to add a Close() method on all components. If that is done, then we can use Close() and Init() for all components and not only bindings and pubsub.

Reply:
> There is work to add a Close() method on all components. If that is done, then we can use Close() and Init() for all components and not only bindings and pubsub.

@artursouza But close means that the components will not be used any more, the logic of suspension might be different due to user's implementation?